### ISSUE TITLE
Hidden files on Mac OS

### ISSUE THREAD

**Issue by rwhite-lightbox** [dastels/circuitpython_deploy#1]
I was loving your deployment scripts until I started to move some of my code into packages:
```
code.py
--common\
      __init__.py
      config.py
      uuid.py
```
On my Macbook, I'm being prompted by the deploy script to manually remove hidden files from my subdir.  This is discussed at adafruit:  https://learn.adafruit.com/welcome-to-circuitpython/troubleshooting

I've found a workaround by changing your script and adding the MacOS option "-X" to your three cp commands.

I'd like to suggest two improvements:
1. detect the OS and add -X for MacOS (darwin).
2. add a "packages" option, similar to your "lib" option to copy local package folders.  Include a packages stanza in the manifest.
Note:  a local Python package feels different than a font or sounds files that might not change as frequently as code.

Your deployment script is a great addition to CircuitPython development, thanks!

**Comment by jliu70** [comment]
I'm not sure if Dave is looking at the GitHub repo.  I recently submitted a PR to update the formatting in the README, but he hasn't responded.  

I'll update my fork to incorporate the change #1 you propose (seems pretty straight forward to import platform and detect platform.system() )

**Comment by rwhite-lightbox** [comment]
@jliu70 Great work on the PR, I made a few comments.  This also fails on Windows and I don't think xcopy is close enough to cp to deal with some of the libs and subs.  I was going to try with robocopy, but would it be better to use Python os and path and skip the system commands all together?

**Comment by jliu70** [comment]
Yes, I haven't tried to run this on Windows.

I have a few laptops I could test on at home.  When I get a chance I can
see if I could extend this to be Windows friendly.

The strange thing about windows is that there are so many variations -
Command Prompt, Powershell, Git bash,  and the new Windows Terminal (aka
WSL2).

**Comment by dastels** [comment]
I try to keep my eye on here, looking at things and approving/merging as soon as I see something.

I a) don't have any Windows machines, and b) don't do dev work on MacOS anymore, so I'm relying on your testing on those platforms.

**Comment by rwhite-lightbox** [comment]
This was merged with #5 some time ago...
